### PR TITLE
Use actual close icon at the top header disclaimer

### PR DESCRIPTION
**Pull request by mmjee**
Ticket: https://team-1624093970686.atlassian.net/browse/AR-736?atlOrigin=eyJpIjoiMjMxNmZjNTYwNWJhNDU2YTkzODU0YjNiMjlhMDAyNWIiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by mmjee** [comment]
The close icon was blue so I copied it and set the fills to #FFFFFF, not sure if that's the right approach...

**Review by shrinathprabhu — APPROVED** [review]
Looks good

**Comment by ajithranka** [comment]
> The close icon was blue so I copied it and set the fills to #FFFFFF, not sure if that's the right approach...

**By creating an icon component**

One possible approach could be turning the SVG icon to a component which takes `color` as a prop.

```
<template>
  <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
    <path d="M1 1L13 13" :stroke="color" stroke-width="2" stroke-linecap="round"/>
    <path d="M13 1L1 13" :stroke="color" stroke-width="2" stroke-linecap="round"/>
  </svg>
</template>

<script>
export default {
  props: {
    color: {
      type: String,
      default: '#13A3FD',
    }
  }
}
</script>
```

**By using `currentColor` and `vite-svg-loader`**

If you prefer avoiding the component, we can also change the hard-coded color values to `currentColor` in the SVG file and set a CSS color on the parent as needed.

```
<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path d="M1 1L13 13" stroke="currentColor" stroke-width="2" stroke-linecap="round"/>
  <path d="M13 1L1 13" stroke="currentColor" stroke-width="2" stroke-linecap="round"/>
</svg>
```

Then inside `AppHeader`, import the SVG file as a component (this will require setting up `vite-svg-loader`).

```
<template>
<span class="banner-close" @click.stop="onCloseBanner" role="button" style="color: #fff">
  <CloseIcon />
</span>
<template>

<script>
// Note: Importing the SVG file as a component not as a path string.
import CloseIcon from '@/assets/iconography/close.svg'
</script>
```

**Review by ajithranka — APPROVED** [review]

File: src/components/AppHeader.vue [hunk 1]
```diff
@@ -77,10 +77,11 @@ import VHeader from "./lib/VHeader/VHeader.vue";
 import { useRoute, useRouter } from "vue-router";
 import { useStore } from "vuex";
 import CancelIcon from "@/assets/iconography/cancel.svg";
+import CloseIcon from "@/components/icons/CloseIcon.vue";
 
 export default {
   name: "AppHeader",
-  components: { VHeader },
+  components: {CloseIcon, VHeader },
```
Comment (ajithranka) [inline src/components/AppHeader.vue:84]: Add consistent spacing between the braces:

```suggestion
  components: { CloseIcon, VHeader },
```

**Review by ajithranka — APPROVED** [review]

**Comment by mmjee** [comment]
Done.

**Pull request merged**